### DISCUSSION TITLE
New npm package

### DISCUSSION THREAD

**Discussion by ghost** [community/community#141577] [npm] [answered]
Hello, everyone.
Today, i made my first npm package for validation. Check it out and give it a star if you like.
https://www.npmjs.com/package/valideasy

**Comment by MorelliniMatteo** [comment, marked as answer]
wow beautiful, i give u  a star but could u make my answer as answered ? i want the badge
 pls hahaha

**Reply by ghost** [reply]
done

**Comment by Ucg2c3** [comment]
I'd like a new Npm I'd give it a star
